JitsiConferenceEvents.js

```diff
@@ -57,6 +57,11 @@ export const CONNECTION_INTERRUPTED = 'conference.connectionInterrupted';
 export const CONNECTION_RESTORED = 'conference.connectionRestored';
 
 /**
+ * A connection to the video bridge's data channel has been established.
+ */
+export const DATA_CHANNEL_OPEN = 'conference.dataChannelOpen';
```

Comment:
I see. I still would prefer to make it consistent with the rest of the events in this class. Even though it's unfortunate we haven't followed the consistency in RTCEvents.